src/vxfw/ScrollBars.zig

```diff
@@ -7,6 +7,7 @@ const Allocator = std.mem.Allocator;
 const ScrollBars = @This();
 
 const vertical_scrollbar_thumb: vaxis.Cell = .{ .char = .{ .grapheme = "▐", .width = 1 } };
+const vertical_scrollbar_hover_thumb: vaxis.Cell = .{ .char = .{ .grapheme = "█", .width = 1 } };
 const horizontal_scrollbar_thumb: vaxis.Cell = .{ .char = .{ .grapheme = "▃", .width = 1 } };
```

Comment:
Let's add a `*_scrollbar_drag_thumb` with `.fg = .{.index = 4}` (copies firefox)

Reply:
Done in https://github.com/rockorager/libvaxis/pull/148/commits/49e1d1d0a7ca607cd68a07eb47c26018e09b13ac 🙌